### PR TITLE
Minor infiltrator fix

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: hippiestation/code/modules/antagonists/infiltrator/items/pinpointer.dm [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 /obj/item/pinpointer/infiltrator
 	name = "infiltration pinpointer"
-	icon = 'hippiestation/icons/obj/device.dmi'
+	icon = 'icons/obj/device.dmi'
```
Comment (yoyobatty) [inline hippiestation/code/modules/antagonists/infiltrator/items/pinpointer.dm]: Changing this fixes the base pinpointer but breaks the upgraded one. Our repo device.dmi fixes the upgraded one but fucks up the arrow.

**Review by yoyobatty — CHANGES_REQUESTED** [review]

File: hippiestation/code/modules/antagonists/infiltrator/items/pinpointer.dm (hunk 1 above)
Comment (ghost) [reply hippiestation/code/modules/antagonists/infiltrator/items/pinpointer.dm]: hmmm, maybe the below fix will work

**Comment by yoyobatty** [comment]
Does it work fully? Like you tested the shit out of it, turned it on and off, upgraded and original?

**Comment by ghost** [comment]
Nope. Lemme do dat

**Comment by ghost** [comment]
@yoyobatty i came up with a lazier solution: adding the arrows to our device.dmi

**Comment by ghost** [comment]
gonna make a broader fix to compensate for the retardeness of hippie players

**Pull request closed without merging**